python/hltTriggerMatch_cfi.py

```diff
@@ -25,23 +25,114 @@
+#Path HLT_Mu50:
+# - first seen online on run 272760 (/cdaq/physics/Run2016/25ns10e33/v1.0/HLT/V16)
+# - last  seen online on run 284044 (/cdaq/physics/Run2016/25ns15e33/v4.2.3/HLT/V2)
+# - V2: (runs 272760 - 274443)
+# - V3: (runs 274954 - 276244)
+# - V4: (runs 276282 - 280385)
+# - V5: (runs 281613 - 284044)
+#Path HLT_TkMu50:
+# - first seen online on run 274954 (/cdaq/physics/Run2016/25ns10e33/v2.1.0/HLT/V14)
+# - last  seen online on run 284044 (/cdaq/physics/Run2016/25ns15e33/v4.2.3/HLT/V2)
+# - V1: (runs 274954 - 275376)
+# - V2: (runs 275656 - 276244)
+# - V3: (runs 276282 - 284044)
+#trigger_paths += ['HLT_Mu50_v%i' % i for i in (2,3,4,5)]
+#overall_prescale = 320 # 196 pb-1
+#overall_prescale = 290  # 1828 pb-1
+
+# http://fwyzard.web.cern.ch/fwyzard/hlt/2017/summary
+#Path HLT_Mu27: 2017
+# - first seen online on run 296070 (/cdaq/physics/Run2017/2e34/v1.0.0/HLT/V2)
+# - last  seen online on run 306460 (/cdaq/physics/Run2017/2e34/v4.2.1/HLT/V2)
+# - V6: (runs 296070 - 297057)
+# - V7: (runs 297099 - 297505)
+# - V8: (runs 297557 - 299329)
+# - V9: (runs 299368 - 299649)
+# - V10: (runs 300079 - 302019)
+# - V11: (runs 302026 - 306171)
+# - V12: (runs 306416 - 306460)
+#Path HLT_Mu50:
+# - first seen online on run 296070 (/cdaq/physics/Run2017/2e34/v1.0.0/HLT/V2)
+# - last  seen online on run 306460 (/cdaq/physics/Run2017/2e34/v4.2.1/HLT/V2)
+# - V6: (runs 296070 - 297057)
+# - V7: (runs 297099 - 297505)
+# - V8: (runs 297557 - 299329)
+# - V9: (runs 299368 - 299649)
+# - V10: (runs 300079 - 302019)
+# - V11: (runs 302026 - 306171)
+# - V12: (runs 306416 - 306460)
+#prescaled_trigger_paths = ['HLT_Mu27_v%i' % i for i in (6, 7, 8, 9, 10, 11)]
+#trigger_paths = ['HLT_Mu50_v%i' % i for i in (6, 7, 8, 9, 10, 11)]
+#overall_prescale = ?
+
+# http://fwyzard.web.cern.ch/fwyzard/hlt/2018/summary
+#Path HLT_Mu27: 2018
+# - first seen online on run 315252 (/cdaq/physics/Run2018/2e34/v1.1.0/HLT/V4)
+# - last  seen online on run 325175 (/cdaq/physics/Run2018/2e34/v3.6.1/HLT/V2)
+# - V12: (runs 315252 - 316271)
+# - V13: (runs 316361 - 325175)
+#Path HLT_Mu50:
+# - first seen online on run 315252 (/cdaq/physics/Run2018/2e34/v1.1.0/HLT/V4)
+# - last  seen online on run 325175 (/cdaq/physics/Run2018/2e34/v3.6.1/HLT/V2)
+# - V12: (runs 315252 - 316271)
+# - V13: (runs 316361 - 325175)
+#Path HLT_OldMu100:
+# - first seen online on run 315252 (/cdaq/physics/Run2018/2e34/v1.1.0/HLT/V4)
+# - last  seen online on run 325175 (/cdaq/physics/Run2018/2e34/v3.6.1/HLT/V2)
+# - V3: (runs 315252 - 325175)
+#Path HLT_TkMu100:
+# - first seen online on run 315252 (/cdaq/physics/Run2018/2e34/v1.1.0/HLT/V4)
+# - last  seen online on run 325175 (/cdaq/physics/Run2018/2e34/v3.6.1/HLT/V2)
+# - V2: (runs 315252 - 325175)
+prescaled_trigger_paths = ['HLT_Mu27_v%i' % i for i in (12,13)]
+trigger_paths = ['HLT_Mu50_v%i' % i for i in (12,13)]
+#trigger_paths += ['HLT_OldMu100_v%i' % i for i in (3)]
+#trigger_paths += ['HLT_TkMu100_v%i' % i for i in (2)]
```

Comment:
doesn't this (and also OldMu100) need to be uncommented?

Reply:
The trigger_paths list for the primary analysis triggers is not actually used in the trigger code anymore. It has been superseded by the lists trigger_filters and trigger_path_names. However, its equivalent prescaled_trigger_paths is used by PrescaleToCommon so it is updated and kept. Also any code that would use trigger_paths is not set up to handle different trigger paths, only different versions of the same single path. The additional commented info from the hlt summary webpages is just for bookkeeping purposes.

Reply:
Is there any critical code that uses trigger_paths?

Reply:
The lepton producer (Zprime2muLeptonProducer_miniAOD) uses it as a backup in case trigger_filters or trigger_path_names are not set, but I don't think anything else uses it.